release-notes/opensearch-release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,606 @@
+* Replace checked-in ml-commons dependency for 2.0 ([#563](https://github.com/opensearch-project/sql/pull/563))
+* Delete ml-commons zip file ([#565](https://github.com/opensearch-project/sql/pull/565))
+* Bump ml-client to 2.0 ([#568](https://github.com/opensearch-project/sql/pull/568))
+
+
+## REFACTORING
+
+### OpenSearch Alerting
+* Remove write Destination APIs ([#412](https://github.com/opensearch-project/alerting/pull/412]))
+* Remove Alerting's notification subproject ([#413](https://github.com/opensearch-project/alerting/pull/413]))
+* Skipping destination migration if alerting index is not initialized ([#417](https://github.com/opensearch-project/alerting/pull/417]))
+* Fix Finding action naming and update release notes ([#432](https://github.com/opensearch-project/alerting/pull/432]))
+
+
+### OpenSearch Alerting Dashboards Plugin
+* Temporarily disabled destination use in some cypress tests to resolve flakiness. ([#214](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/214))
+* Remove disabled buttons and update Destination flows to reflect read-only state ([#221](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/221))
+ING
+
+
+### OpenSearch Common Utils
+* Remove feature and feature_list usage for Notifications ([#136](https://github.com/opensearch-project/common-utils/pull/136))
+* Rename references for Get Channels API for Notifications ([#140](https://github.com/opensearch-project/common-utils/pull/140))
+* Remove allowedConfigFeatureList from GetPluginFeaturesResponse for Notifications ([#144](https://github.com/opensearch-project/common-utils/pull/144))
+* Remove NotificationEvent Request, Response and SearchResults ([#153](https://github.com/opensearch-project/common-utils/pull/153))
+* Add NotificationEvent to SendNotificationResponse and Removal of NotificationID ([#156](https://github.com/opensearch-project/common-utils/pull/156))
+
+
+### OpenSearch k-NN
+* Refactor KNNCodec to use new extension point ([#319](https://github.com/opensearch-project/k-NN/pull/319))
+* Refactor BWC tests into sub project ([#359](https://github.com/opensearch-project/k-NN/pull/359))
+
+
+### OpenSearch Ml Commons
+* Removed RCF jars and updated to fetch RCF 3.0-rc2 from maven ([#277](https://github.com/opensearch-project/ml-commons/pull/277))
+
+
+### OpenSearch Observability
+* Modularize and cleanup traces ([#601](https://github.com/opensearch-project/observability/pull/601))
+* Modularize and cleanup panel ([#603](https://github.com/opensearch-project/observability/pull/603))
+* Modularize event Analytics live tail and fix bug ([#647](https://github.com/opensearch-project/observability/pull/647))
+* Fix lint and modularize dashboard ([#583](https://github.com/opensearch-project/observability/pull/583))
+* Modularize service and fix issues ([#595](https://github.com/opensearch-project/observability/pull/595))
+
+
+## Deprecation
```

Comment:
If we have this heading we should be putting the all the inclusive language stuff here too.

Reply:
This section is removed as PA moved this to `Documentation`. 
https://github.com/opensearch-project/performance-analyzer/pull/196